spec/controllers/users/two_factor_authentication_controller_spec.rb

```diff
@@ -81,6 +82,20 @@ def index
 
         expect(response).to redirect_to login_two_factor_piv_cac_path
       end
+
+      it 'redirects to phone when on mobile and user has phone' do
+        allow_any_instance_of(Browser).to receive(:mobile?).and_return(true)
```

Comment:
could we stub `controller.mobile?` here instead?
```suggestion
        allow(controller).to receive(:mobile?).and_return(true)
```